### PR TITLE
add whatsnew popup to drop ledger support for firefox

### PR DESCRIPTION
**Pull request by montelaidev**
## Explanation

Notice to drop support ledger support for firefox and encourage users to switch to chrome.

### After
image

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [x] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by HowardBraham** [comment]
All the code looks good, except I think the coverage on `shared/notifications/index.js` and `ui/components/app/whats-new-popup/whats-new-popup.js` is so low that it's critically failing Codecov

**Review by danjm — DISMISSED** [review]

**Review by HowardBraham — DISMISSED** [review]

**Review by HowardBraham — DISMISSED** [review]

**Review by HowardBraham — DISMISSED** [review]

**Comment by HowardBraham** [comment]
After hitting merge conflicts on this PR two separate times, I'm trying a new experimental strategy to reduce future merge conflicts for Notifications.  (I may also apply this to Migrations later).

I define `export const NOTIFICATION_DROP_LEDGER_FIREFOX = 25`
and then use the syntax
```
export const UI_NOTIFICATIONS = {
...
  [NOTIFICATION_DROP_LEDGER_FIREFOX]: {
  },
};
```

So there's only one single place in the code that this Notification is defined to be 25.  If someone merges another Notification 25 first, I will still have to resolve the adjacent line changes, but I only have to change **one** 25 to a 26.

Let me know what you think.

**Review by HowardBraham — APPROVED** [review]

File: shared/notifications/index.js [hunk 1]
```diff
@@ -135,6 +142,11 @@ export const UI_NOTIFICATIONS = {
     id: 24,
     date: null,
   },
+  // This syntax is unusual, but very helpful here.  It's equivalent to `UI_NOTIFICATIONS[NOTIFICATION_DROP_LEDGER_FIREFOX] =`
+  [NOTIFICATION_DROP_LEDGER_FIREFOX]: {
+    id: Number(NOTIFICATION_DROP_LEDGER_FIREFOX),
+    date: null,
+  },
```
Comment (gantunesr) [inline shared/notifications/index.js:149]: @montelaidev you want to add the notification number in here?

**Review by gantunesr — APPROVED** [review]
LGTM

File: shared/notifications/index.js (hunk 1 above)
Comment (gantunesr) [reply shared/notifications/index.js:149]: As a comment

Comment (HowardBraham) [reply shared/notifications/index.js:149]: Yeah I think I was also confused about where he meant to add it.  Adding it here would increase the merge conflict work.

**Comment by plasmacorral** [comment]
Confirmed that this message only appears for users with Ledger present on Firefox when testing migration from 10.30 to the feature branch in both Chrome 116 and Firefox 110 on Mac Ventura 13.5.1.

**Pull request merged**